portuguese-pt.ini

```
@@ -0,0 +1,661 @@
+﻿[Header]
+Language = "Português (PT)"
+Author = "peachflavored"

```

Missing the 'Author2' string.
```suggestion
Author = "peachflavored"
Author2 = ""
```

---

You should add `Katherine_S2003` for 'Author' as well, since she contributed to it.

---

This has been fixed. Katherine_S2003 added to "Author2".

---

> 
> 
> Katherine_S2003 should be added to "Author" instead
> See the picture in the issue below, "Author2" is used for that newline where "unknown, Fedprod" is seen. This can be observed here as well
> 
> https://github.com/AM2R-Community-Developers/AM2R-Localizations/blob/53d9c5dbb074fc5825ddfb73585f3b5ce2574d5a/german.ini#L4

Fixed.